classes/Tools.php

```diff
@@ -2321,6 +2321,44 @@ public static function getMediaServer($filename)
         return Tools::usingSecureMode() ? Tools::getShopDomainSsl() : Tools::getShopDomain();
     }
 
+    /**
+     * Get domains information
```

Comment:
Would it be possible to have more details on the array content?

Reply:
Is it better? I don't know what to say about this method :/